tests/security/cc/audit_remote_libvirt.pm

```diff
@@ -29,7 +29,7 @@ sub run {
     assert_script_run('systemctl start libvirtd');
     assert_script_run('virsh net-define /etc/libvirt/qemu/networks/default.xml');
     assert_script_run('virsh net-start default');
-    is_sle('15-sp6+') ?
+    is_sle('15-sp5+') ?
```

Comment:
@alice-suse

Reply:
AFAIK, modular libvirt daemons are provided on 15sp5, but not the default one. The default is monolithic libvirtd. Does any recent change modify this? BTW, I think this can be confirmed on a fresh installation to see if `systemctl status libvirtd.service` is active.

If the default has really switched to modular libvirt daemons, then the following line 34 ` assert_script_run('systemctl is-active libvirtd');`  shall NOT be here, because libvirtd service is not active any more if virtqemud modular daemon is active.

Reply:
Are you saying that changes in #18110 are also incorrect and should be modified?

Reply:
The behavior has changed on SP5 in build [135.14](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP5&build=135.14) a month ago.

Reply:
Sorry my bad, I did not read it correctly.

I had a check in the failure job's [log](https://openqa.suse.de/tests/13606551/file/audit_remote_libvirt-systemctl_status.log) , and get below. I think from this, it can be confirmed that modular libvirt daemon is running --virt{qemu,netword}d.service.

```
             ├─virtqemud.service 
             │ └─ 17076 /usr/sbin/virtqemud --timeout 120
             ├─systemd-journald.service 
             │ └─ 939 /usr/lib/systemd/systemd-journald
             ├─sshd.service 
             │ └─ 1811 sshd: /usr/sbin/sshd -D [listener] 0 of 10-100 startups
             ├─wickedd-dhcp4.service 
             │ └─ 1109 /usr/lib/wicked/bin/wickedd-dhcp4 --systemd --foreground
             ├─systemd-machined.service 
             │ └─ 1103 /usr/lib/systemd/systemd-machined
             ├─systemd-hostnamed.service 
             │ └─ 17312 /usr/lib/systemd/systemd-hostnamed
             ├─virtnetworkd.service 
             │ ├─ 17098 /usr/sbin/virtnetworkd --timeout 120
             │ ├─ 17200 /usr/sbin/dnsmasq --conf-file=/var/lib/libvirt/dnsmasq/default.conf --leasefile-ro -
```


So I'd suggest below:

```
    is_sle('15-sp5+') ?  restart_modular_libvirt_daemons :assert_script_run('systemctl start libvirtd');
    assert_script_run('virsh net-define /etc/libvirt/qemu/networks/default.xml');
    assert_script_run('virsh net-start default');
    is_sle('15-sp5+') ?
      check_modular_libvirt_daemons('qemu') :    # when system uses modular libvirt daemons
      assert_script_run('systemctl is-active libvirtd');    # when system uses monolithic libvirt daemons
```

Reply:
Ok, I implemented your suggestions and it seems to be passing: https://openqa.suse.de/tests/13902708
Again, the fail is caused by a different issue.

Reply:
Based on the latest verification job output at https://openqa.suse.de/tests/13902708#step/audit_remote_libvirt/40, it seems needed to run `systemctl daemon-reload` before `restart_modular_libvirt_daemons`.

Reply:
Looking good now: https://openqa.suse.de/tests/13924430#step/audit_remote_libvirt/42